arches_controlled_lists/src/arches_controlled_lists/components/tree/ListTree.vue

```diff
@@ -93,25 +99,38 @@ const navigate = (newRoute: RouteLocationNormalizedLoadedGeneric) => {
             if (!tree.value.length) {
                 return;
             }
-            const { found, path } = findNodeInTree(
-                tree.value,
-                newRoute.params.id as string,
-            );
-            if (found) {
-                setDisplayedRow(found.data);
-                const itemsToExpandIds = path.map(
-                    (itemInPath: TreeNode) => itemInPath.key,
+            try {
+                const { found, path } = findNodeInTree(
+                    tree.value,
+                    newRoute.params.id as string,
                 );
-                expandedKeys.value = {
-                    ...expandedKeys.value,
-                    ...Object.fromEntries(
-                        [
-                            found.data.controlled_list_id,
-                            ...itemsToExpandIds,
-                        ].map((x) => [x, true]),
+
+                if (found) {
+                    setDisplayedRow(found.data);
+                    const itemsToExpandIds = path.map(
+                        (itemInPath: TreeNode) => itemInPath.key,
+                    );
+                    expandedKeys.value = {
+                        ...expandedKeys.value,
+                        ...Object.fromEntries(
+                            [
+                                found.data.controlled_list_id,
+                                ...itemsToExpandIds,
+                            ].map((x) => [x, true]),
+                        ),
+                    };
+                    selectedKeys.value = { [found.data.id]: true };
+                }
+            } catch (error) {
```

Comment:
Should we show toast to alert user that the URL they navigated to did not result in an item hit?

Reply:
I think so 👍